web/modules/custom/demo_users/demo_users.module

```diff
@@ -33,7 +33,7 @@ function demo_users_rdf_entity_insert(EntityInterface $entity) {
 
   // Check if there is already a membership. If one of the facilitators create
   // an entity, then the membership will already exist.
-  if (!Og::getMembership($entity, $facilitator)) {
+  if (!Og::getMembership($entity, $facilitator) && ($facilitator->id() != \Drupal::currentUser()->id())) {
```

Comment:
Can you drop a line to explain this? From what I can understand, we skip creating the relationship when the facilitator is the current user because OG will handle this already, right?